hail/src/test/scala/is/hail/backend/spark/SparkBackendSuite.scala

```diff
@@ -0,0 +1,43 @@
+package is.hail.backend.spark
+
+import is.hail.SparkSuite
+import is.hail.expr.ir
+import is.hail.utils._
+import org.apache.spark.sql.Row
+import org.testng.annotations.Test
+
+class SparkBackendSuite extends SparkSuite {
+
+  @Test def testRangeCount() {
+    val node = ir.ApplyBinaryPrimOp(ir.Add(), ir.TableCount(ir.TableRange(10, 2)), ir.TableCount(ir.TableRange(15, 5)))
+    assert(SparkBackend.execute(hc.sc, node, optimize = false) == 25)
```

Comment:
So these should be just be TableIR tests in TableIRSuite.  You should call assertEvalsTo, etc., but those functions should run the SparkBackend (when it implements the pipeline) and compare results.  We don't want to duplicate the set of IR functionality for each backend, but have a broad set of tests that verify standard behavior and all backends should be tested by those tests.

You could also have testing functions for TableIR that collects the results and compares them.